bldsys/cmake/component_helpers.cmake

```diff
@@ -16,21 +16,35 @@
 
 add_custom_target(vkb_components)
 
+set_property(TARGET vkb_components PROPERTY FOLDER "components")
```

Comment:
What is that custom target `vkb_components` for? Is it just to have one central target for all components?
(The same question holds with `vkb_tests`.)